package.json

```diff
@@ -77,7 +77,7 @@
     "lodash.debounce": "^4.0.8",
     "lowlight": "^3.1.0",
     "lucide-react": "^0.292.0",
-    "next": "14.0.3",
+    "next": "14.2.6",
```

Comment:
Is there a need to update the Next.js version for this change? Please confirm if the current version (`14.0.3`) supports all the features we're implementing or if an upgrade is necessary.